### DISCUSSION TITLE
occ scan path argument

### DISCUSSION THREAD

**Discussion by plrnr** [nextcloud/all-in-one#809] [Questions] [answered]
What do I pass as an argument to occ scan 
sudo docker exec nextcloud-aio-nextcloud php occ files:scan --path=

I am getting the job done by specifying --all to the above command instead, but it is costly. What path do I specify to scan a particular directory where I copied files manually and I would like to scan so nextcloud db can sync up.

**Comment by szaimen** [comment]
Hi, please ask this in the forums as it is a general Nextcloud occ question.

**Reply by plrnr** [reply]
hi Simon,

I know how to specify a path generally with nextcloud installed directly on the os. I am asking specifically for AIO.
Let me try again..

1. Is nextcloud-aio-nextcloud the right container where files are accessible?
2. If so, the command below doesn't return anything
`sudo docker exec nextcloud-aio-nextcloud bash -c "find /var/www/html/ -name 'IMG_8621.mp4'"`

I am basically converting MOV files to mp4 so a browser can play videos. Now, after converting I need to perform a scan to update nextcloud db.

**Comment by szaimen** [comment, marked as answer]
Here is how to run occ commands: https://github.com/nextcloud/all-in-one#how-to-run-occ-commands and here is the documentation on the scan command: https://docs.nextcloud.com/server/23/admin_manual/configuration_server/occ_command.html?highlight=scan#scan

**Reply by plrnr** [reply]
Thanks a lot. This worked - "user_id/files/Photos"
